docs/source/algorithms/MonteCarloAbsorption-v1.rst

```diff
@@ -112,6 +112,7 @@ When the sparse instrument option is enabled, a sparse instrument corresponding
 The interpolation is a two step process: first a spatial interpolation is done from the detector grid of the sparse instrument to the actual detector positions of the full instrument. Then, if `ResimulateTracksForDifferentWavelengths` = True the correction factors are interpolated over the missing wavelengths.
 
 .. note:: Currently, the sparse instrument mode does not support instruments with varying *EFixed*.
+.. note:: Errors are not currently provided for absorption correction factors calculated involving spatial interpolation
```

Comment:
Is there a fundamental reason for this?
Would it makes sense to interpolate also the errors?

Reply:
I think we should extend this to calculate the errors for interpolated points also. The only reason I've not done it here was that it seemed a relatively large piece of work so I've created another issue for it (#29000). I think the error calculation for the interpolated points would include two contributions: 
a) propagate errors from the simulated points
b) include a contribution from the "interpolation error" ie how well does the interpolation function match the actual one

Reply:
My guess is that b) could be ignored if you have high order splines. The corrections are supposed to spatially smooth (except for the flat plate at the sample angle), but there correction is problematic anyhow, so one should be careful with using spare instrument in that case. Agreed that a) can be done in a separate PR and the gross approximation would be just to run the same spline interpolation on the 'errors'.

Reply:
OK good. It was estimating the interpolation error that seemed the hardest part. I was hoping that the gsl library we use for the spline interpolation returned the interpolation error but surprisingly it doesn't.

Reply:
I knew it doesn't since the exact same problem is there for SplineInterpolation algorithm; no errors at the moment.